examples/quantization_aware_training/torch/anomalib/main.py

```diff
@@ -0,0 +1,161 @@
+    xml_size = ir_path.stat().st_size
+    bin_size = ir_path.with_suffix(".bin").stat().st_size
+    for t in ["bytes", "Kb", "Mb"]:
+        if m_type == t:
+            break
+        xml_size /= 1024
+        bin_size /= 1024
+    model_size = xml_size + bin_size
+    print(f"Model graph (xml):   {xml_size:.3f} Mb")
+    print(f"Model weights (bin): {bin_size:.3f} Mb")
+    print(f"Model size:          {model_size:.3f} Mb")
+    return model_size
+
+
+def main():
+    ###############################################################################
+    # Step 1: Prepare the model and dataset
+    print(os.linesep + "[Step 1] Prepare the model and dataset")
+
+    model = Stfpm()
+    datamodule = MVTec(root=DATASET_PATH)
+
+    # Create an engine for the original model
+    engine = Engine(task=TaskType.SEGMENTATION, default_root_dir=FP32_RESULTS_ROOT)
+    if USE_PRETRAINED:
+        # Load the pretrained checkpoint
+        CHECKPOINT_PATH.mkdir(parents=True, exist_ok=True)
+        ckpt_path = CHECKPOINT_PATH / "model.ckpt"
+        torch.hub.download_url_to_file(CHECKPOINT_URL, ckpt_path)
+    else:
+        # (Optional) Train the model from scratch
+        engine.fit(model=model, datamodule=datamodule)
+        ckpt_path = engine.trainer.checkpoint_callback.best_model_path
+
+    print("Test results for original FP32 model:")
+    fp32_test_results = engine.test(model=model, datamodule=datamodule, ckpt_path=ckpt_path)
+
+    ###############################################################################
+    # Step 2: Quantize the model
+    print(os.linesep + "[Step 2] Quantize the model")
+
+    # Create calibration dataset
+    def transform_fn(data_item):
+        return data_item["image"]
+
+    test_loader = datamodule.test_dataloader()
+    calibration_dataset = nncf.Dataset(test_loader, transform_fn)
+
+    # Quantize the inference model using Post-Training Quantization
+    inference_model = model.model
+    quantized_inference_model = nncf.quantize(model=inference_model, calibration_dataset=calibration_dataset)
+
+    # Deepcopy the original model and set the quantized inference model
+    quantized_model = deepcopy(model)
+    quantized_model.model = quantized_inference_model
+
+    # Create engine for the quantized model
+    engine = Engine(task=TaskType.SEGMENTATION, default_root_dir=INT8_RESULTS_ROOT, max_epochs=1)
+
+    # Validate the quantized model
+    print("Test results for INT8 model after PTQ:")
+    int8_init_test_results = engine.test(model=quantized_model, datamodule=datamodule)
+
+    ###############################################################################
+    # Step 3: Fine tune the quantized model
+    print(os.linesep + "[Step 3] Fine tune the quantized model")
+
+    engine.fit(model=quantized_model, datamodule=datamodule)
+    print("Test results for INT8 model after QAT:")
+    int8_test_results = engine.test(model=quantized_model, datamodule=datamodule)
```

Comment:
Please print result of validation here as it done in resnet18 example https://github.com/openvinotoolkit/nncf/blob/develop/examples/quantization_aware_training/torch/resnet18/main.py#L270

Reply:
`engine.test` prints results the following way: 
![image](https://github.com/openvinotoolkit/nncf/assets/3229971/8ffd8ef2-ed88-4fb7-8293-7b5de9c27ff7)